CRM/Report/Form.php

```diff
@@ -1574,12 +1580,35 @@ public function addOptions() {
    */
   public function addChartOptions() {
     if (!empty($this->_charts)) {
-      $this->addElement('select', "charts", ts('Chart'), $this->_charts);
+      $this->addElement('select', "charts", ts('Chart'), $this->getChartLabels($this->_charts));
```

Comment:
@braders your passing through $this->_charts as a function param but the function in L1594 doesn't accept parameters